### DISCUSSION TITLE
Unable to view updates when pushing to existing or creating new branches

### DISCUSSION THREAD

**Discussion by garrettApproachableGeek** [community/community#57443] [Repositories]
### Select Topic Area

Question

### Body

We are unable to push to an existing branch on origin or create a new branch.
Multiple people are having the same issue on our team
Seems like there is some degraded service on Github showing updates to remote

`git push -u origin new-branch`
`git push -u origin existing-branch`

This is the private repo it is happening on:
https://github.com/Approachable-Geek/geek-website

**Comment by nstanard** [comment]
This was happening to me as well until just a moment ago. down detection shows lots of reports the last few min: https://downdetector.com/status/github/
